### PR TITLE
React children

### PR DESCRIPTION
**Pull request by jessieengstrom**
- deleted App.js
- added links to WebsiteLayout component header (about, order, contact)
- deleted Button.js
- Deleted Update.js
- Added  routes contact, update, about, order to router

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +0,0 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
```
Comment (a15n) [inline src/App.js]: let's also delete App.css and logo.svg

Comment (a15n) [inline src/App.js]: let's also delete App.css and logo.svg

File: src/components/Button.js [hunk 2]
```diff
@@ -1,16 +0,0 @@
-import React from 'react';
```
Comment (a15n) [inline src/components/Button.js]: let's keep this. We aren't using it now but will use it later

File: src/App.js (hunk 1 above)
Comment (jessieengstrom) [reply src/App.js]: Added Button.js back and removed App.css and logo.svg

**Pull request merged**